README.md

```diff
@@ -1,19 +1,19 @@
-# Docker ACI Integration (Beta)
+# Docker Cloud Integration (Beta)
 
 > :warning: **This CLI is in beta**: The installation process, commands, and
 > flags will change in future releases.
 
-Docker has extended its strategic collaboration with Microsoft to simplify code
+Docker has extended its strategic collaboration with Microsoft and Amazon to simplify code
 to cloud application development for developers and development teams by more
-closely integrating with Azure Container Instances (ACI).
+closely integrating with Azure Container Instances (ACI) and Amazon EC2 Container Service (ECS).
 
-The new ACI experience from local VS Code and Docker Desktop development to
-remote deployment in ACI creates a tighter integration between Docker and
-Microsoft developer technologies provides the following productivity benefits to
+The new Cloud experience from local VS Code and Docker Desktop development to
+remote deployment in ACI/ECS creates a tighter integration between Docker and
```

Comment:
Until we have Compose in the API or run support for ECS, the API won't work with ECS